### PR TITLE
[#322] Show objects data as JSON in the admin interface.

### LINKED ISSUE #322: Objects API admin: Show Object data as JSON
When viewing the Object data in the admin, it is quite annoying that the data is shown as a Python dict instead of (what you would expect) a JSON blob:

image

I would recommend to show the Objects as JSON, this to ease copy-pasting data and making this consistent with the ObjectTypes (which are shown as JSON)

### PR DESCRIPTION
**Pull request by ErhanCitil**
fixes #322 

afbeelding

### CODE REVIEW

**Comment by ErhanCitil** [comment]
@alextreme eindelijk groen!

**Comment by alextreme** [comment]
@ErhanCitil hiervoor zouden geen additionele relaties nodig zijn in token/models.py https://github.com/maykinmedia/objects-api/pull/331/files#diff-539f45b9dfaed2de7f65e9a75a243d0b1bf347c77cc802b2888fc0bf57dd110e

**Comment by ErhanCitil** [comment]
@alextreme deze kan ook nagekeken worden

**Comment by ErhanCitil** [comment]
@alextreme rebase voltooid

File: src/objects/token/migrations/0009_auto_20230519_1105.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.28 on 2023-05-19 09:05
```
Comment (alextreme) [inline src/objects/token/migrations/0009_auto_20230519_1105.py:1]: Please remove this migration

File: src/objects/token/migrations/0010_auto_20230612_1015.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.28 on 2023-06-12 08:15
```
Comment (alextreme) [inline src/objects/token/migrations/0010_auto_20230612_1015.py:1]: Please remove this migration

**Comment by annashamray** [comment]
It's hard to say but it looks like this PR includes changes from  PR https://github.com/maykinmedia/objects-api/pull/341 

I'll review it when the PR with dependencies and linting is merged and this one is rebased on it.

**Pull request closed without merging**